src/classes/list-object.ts

```diff
@@ -0,0 +1,34 @@
+import { NS_RDF } from '../constants'
+
+export class ListObject {
+  s: string
+  p: string
+  graph: any
+  termFactory: any
+  NS_Rdf: string
+
+  constructor(s: string, p: string, graph: any, termFactory: any) {
```

Comment:
Can we clean up this constructor a bit? Maybe using this pattern? https://kendaleiv.com/typescript-constructor-assignment-public-and-private-keywords/